src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -11,7 +11,11 @@
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.logic.parser.AddressBookParser;
 import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.ElisaState;
+import seedu.address.model.ElisaStateHistory;
+import seedu.address.model.ElisaStateManager;
 import seedu.address.model.ItemModel;
+import seedu.address.model.ItemModelManager;
```

Comment:
I think at this moment we shouldn't expose the ItemModelManager. Perhaps we should move the method that you need, `addToSeparateList` to the interface.